blog/app/Http/Controllers/ErrorController.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class ErrorController extends Controller
+{
+    public function error($request){
+        return $request->message;
+    }
+}
```

Comment:
Ne znam zasta ga koirstis, ali nikad neces imati error controller, jer uvijek ces neki error dobijati preko kontrolera koji zapravo ima neku funkcionalnost, da li ce to biti 500 ukoliko pukne BE , 401 za Unautorized ili 404 ukoliko ne nadje rutu itd...